cmd/runtimetest/main.go

```diff
@@ -271,6 +271,62 @@ func validateDefaultFS(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateLinuxDevices(spec *rspec.Spec) error {
+	logrus.Debugf("validating linux devices")
+
+	for _, device := range spec.Linux.Devices {
+		fi, err := os.Stat(device.Path)
+		if err != nil {
+			return err
+		}
```

Comment:
On Wed, Sep 14, 2016 at 02:42:37AM -0700, Ma Shimiao wrote:
> > We can address that when we switch from the current “lots of tests
> > run from within one container” to a test harness that lets us run
> > “lots of little one-test containers”.
>
> That means we need to change the whole test structure

I think that is unavoidable [1,2].  But you can certainly rewrite
runtimetest to not carry on instead of dying on errors if you want to
finish the whole testcase [3].

[1]: https://github.com/opencontainers/ocitools/pull/61#issuecomment-235343447
[2]: https://github.com/opencontainers/ocitools/issues/66#issuecomment-235753741
[3]: https://github.com/opencontainers/ocitools/pull/211/files#r78707264